index.js

```diff
@@ -29,7 +29,9 @@ module.exports = function promisify(fn, ctx, _) {
   }
 
   return function promise() {
-    var args = slice.call(arguments, 0, Math.max(argsNum - argsToPrepend.length, 0));
+    for (var args = Array(Math.max(argsNum - argsToPrepend.length, 0)), i = 0; i < args.length; ++i) {
```

Comment:
`Math.max(argsNum - argsToPrepend.length, 0)` is constant, you can move this from function